mcs/class/Mono.Http/samples/auth.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- This file can be used together with authentication modules -->
 <users>
-<user name="gonzalo" password="lalala">
+<user name="gonzalo" password="PLACEHOLDER">
```

Comment:
does it not like [PLACEHOLDER] ?  --- applicable here and all other changes

Reply:
I wasn't sure if ['s can go in sql connection strings (as elsewhere in this PR), but it looks like that should be ok.